tuxemon/config.py

```diff
@@ -88,6 +88,8 @@ def __init__(self, config_path: Optional[str] = None) -> None:
         self.dev_tools = cfg.getboolean("game", "dev_tools")
         self.recompile_translations = cfg.getboolean("game", "recompile_translations")
         self.skip_titlescreen = cfg.getboolean("game", "skip_titlescreen")
+        self.compress_save = cfg.getboolean("game", "compress_save")
```

Comment:
what happens if both are true?  might be better to make is a string value, so that "gzip", or "lzma" can be specified